tests/testthat/test-compiler-animation.R

```diff
@@ -1,15 +1,20 @@
+library(data.table)
 acontext("animation")
 
-if(require(maps) && require(plyr)){
+if(require(maps)){
   data(UStornadoes, package = "animint2")
   stateOrder <- data.frame(state = unique(UStornadoes$state)[order(unique(UStornadoes$TornadoesSqMile), decreasing=T)], rank = 1:49) # order states by tornadoes per square mile
   UStornadoes$state <- factor(UStornadoes$state, levels=stateOrder$state, ordered=TRUE)
   UStornadoes$weight <- 1/UStornadoes$LandArea
   # useful for stat_bin, etc.
   USpolygons <- map_data("state")
-  USpolygons$state = state.abb[match(USpolygons$region, tolower(state.name))]
-  UStornadoCounts <-
-    ddply(UStornadoes, .(state, year), summarize, count=length(state))
+  USpolygons$state <- state.abb[match(USpolygons$region, tolower(state.name))]
+
+  # Creating a copy of UStornadoes using base R
+  UStornadoes_copy <- data.frame(UStornadoes)
+  setDT(UStornadoes_copy)  # Converting the copy to data.table
```

Comment:
please use `UStornadoes_dt <- data.table(UStornadoes)` which is simpler and easier to understand